scorer/runner.py

```diff
@@ -1,24 +1,36 @@
+import os
 import time
+import zipfile
 import socket
 import traceback
-import os
 from datetime import datetime
 from arango import ArangoClient
+from py_expression_eval import Parser
+from hashlib import sha256
+import base64
 import config
+from verifications import yekta
 from verifications import seed_connected
 from verifications import brightid
+from verifications import seed_connected
 from verifications import dollar_for_everyone
-from verifications import yekta
 from verifications import seed_connected_with_friend
-from py_expression_eval import Parser
 
 db = ArangoClient(hosts=config.ARANGO_SERVER).db('_system')
+snapshot_db = ArangoClient().db('snapshot')
```

Comment:
Update all `ArangoClient()` to use `hosts=config.ARANGO_SERVER`

Reply:
It's done.